nodeadm/internal/containerd/config.go

```diff
@@ -4,18 +4,21 @@ import (
 	"bytes"
 	"context"
 	_ "embed"
+	"path/filepath"
 	"text/template"
 
 	"github.com/aws/aws-sdk-go-v2/feature/ec2/imds"
 	"github.com/awslabs/amazon-eks-ami/nodeadm/internal/api"
 	"github.com/awslabs/amazon-eks-ami/nodeadm/internal/util"
+	"go.uber.org/zap"
 )
 
 const ContainerRuntimeEndpoint = "unix:///run/containerd/containerd.sock"
 
 const (
-	containerdConfigFile = "/etc/containerd/config.toml"
-	containerdConfigPerm = 0644
+	containerdConfigFile      = "/etc/containerd/config.toml"
+	containerdConfigImportDir = "/etc/containerd/config.d"
```

Comment:
how will this drop-in been pickup by containerd?
And is there any override to the config.toml?

Reply:
the required change in the base containerd config happened previously so that's why u don't see it here 😅 
https://github.com/awslabs/amazon-eks-ami/blob/e5426f60dea1dcf0256ba5a3d249ce0fd7e892d8/nodeadm/internal/containerd/config.template.toml#L4-L7

Reply:
no changes happen to the config.toml itself